setup.cfg

```diff
@@ -43,7 +43,7 @@ install_requires =
     dataclasses; python_version < "3.7.0"
 python_requires = >=3.6.1
 
-[options.data_files]
+[data_files]
```

Comment:
This change is wrong. All the sections with options must be prefixed with `options.` for setuptools to find them.